src/gui/mainwindow.cpp

```diff
@@ -805,6 +807,12 @@ void MainWindow::displayRSSTab() const
         m_tabs->setCurrentWidget(m_rssWidget);
 }
 
+void MainWindow::displayExecutionLogTab() const
+{
```

Comment:
Yes, but don't understand why do you still need the the test for `m_executionLog`.